### DISCUSSION TITLE
Improved chat tweaks

### DISCUSSION THREAD

**Discussion by IdotIcom** [BedrockTweaks/Files#259] [Ideas]
Make "Chat tweaks" function like this!!

https://github.com/user-attachments/assets/8962a89e-f944-4ba3-85d7-ff7430c7df30

🙏🙏

**Comment by SquatchHunter** [comment]
from what I saw on the discord discussion this will not be possible with just an RP and within the client

do correct if wrong

**Comment by IdotIcom** [comment]
There are 3 RP that I know have this feature 

Better bedrock
https://betterbedrock.com/

Endd Better UI
https://mcpedl.com/endd-better-ui-pack/


Gote-Supporter
https://mcpedl.com/gote-supporter/
